### PR TITLE
Docs: using Podman to run Metabase

### PR DESCRIPTION
**Pull request by ariya**
This change is [Reviewable](https://reviewable.io/reviews/metabase/metabase/27154)

### CODE REVIEW

**Comment by jeff-bruemmer** [comment]
@ariya are you still working on this PR? Or is it ready to go?

**Comment by ariya** [comment]
> @ariya are you still working on this PR? Or is it ready to go?

There are few more things I plan to do.
However, do you think this is in the right direction @jeff-bruemmer ?

**Comment by jeff-bruemmer** [comment]
The instructions are easy to follow, with easy-to-copy-code. There's some repetition, though; you could consider consolidating the two sections, or I could follow up once you finish the draft.

**Comment by ariya** [comment]
> There's some repetition, though; you could consider consolidating the two sections, or I could follow up once you finish the draft.

The repetition is originated from the Docker instructions (`running-metabase-on-docker.md`), which basically serves as the basis as this Podman flavor.

It makes sense to reduce the repetitions, perhaps once this Podman PR is merged.

**Comment by ariya** [comment]
@jeff-bruemmer This is now ready for review. Any further improvements can be in follow-up PRs.

File: docs/installation-and-operation/installing-metabase.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ The simplest and most basic way of running Metabase.
 
 If you prefer to use a Docker container, we've got you covered.
 
+## [Running on Podman](running-metabase-on-podman.md)
+
+Our Docker image is also compatible with Podman, if that is what you prefer.
```
Comment (jeff-bruemmer) [inline docs/installation-and-operation/installing-metabase.md:23]:
```suggestion
Our Docker image is also compatible with Podman.
```

**Review by jeff-bruemmer — APPROVED** [review]
Looks good. Thanks, @ariya!

**Pull request merged**